src/IIIFPresentation/API/API.csproj

```diff
@@ -27,6 +27,9 @@
         <Content Include="..\.dockerignore">
             <Link>.dockerignore</Link>
         </Content>
+        <Content Update="appsettings.Development.json">
```

Comment:
This isn't checked in - should it be?

Reply:
this can contain dev settings like connection strings, so should be around locally, but I don't think it should be checked in